astropy/units/physical.py

```diff
@@ -311,6 +313,19 @@ def __init__(self, unit, physical_types):
     def __iter__(self):
         yield from self._physical_type_list
 
+    def __getattr__(self, attr):
+        if attr in dir(str):
+            warning_message = (
+                f"support for accessing str attributes such as {attr} "
+                "from PhysicalType instances is deprecated since 4.3 and will be removed in a "
+                "subsequent release.")
+            warnings.warn(warning_message, AstropyDeprecationWarning)
+            self_str_attr = getattr(str(self), attr, None)
+            return self_str_attr
+        else:
+            raise AttributeError(
```

Comment:
Just do `super().__getattribute__(attr)` so you get the standard error message.

Reply:
🤣. @namurphy, apologies for suggesting otherwise.